Yoti.Auth.Sandbox.Tests/DocScan/DocScanSandboxClientTests.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using Moq;
+using Xunit;
+using Yoti.Auth.Sandbox.DocScan.Request;
+using Yoti.Auth.Tests.Common;
+
+namespace Yoti.Auth.Sandbox.Tests.DocScan
+{
+    public class DocScanSandboxClientTests
+    {
+        private const string _someSdkId = "someSdkId";
+        private const string _someSessionId = "someSessionId";
+        private readonly SandboxClient _yotiDocScanSandboxClient;
+        private static readonly Uri _someUri = new Uri("https://www.test.com");
```

Comment:
Could we have a test checking the default production URL?

Reply:
Added in 6416219